src/picongpu/include/particles/Particles.tpp

```diff
@@ -167,13 +167,19 @@ void Particles<T_ParticleDescription>::update(uint32_t )
         typename GetFlagType<FrameType,interpolation<> >::type
         >::type InterpolationScheme;
 
-    typedef typename GetMargin<InterpolationScheme>::LowerMargin LowerMargin;
-    typedef typename GetMargin<InterpolationScheme>::UpperMargin UpperMargin;
+    typedef typename GetMargin<InterpolationScheme>::LowerMargin InterpolationLowerMargin;
+    typedef typename GetMargin<InterpolationScheme>::UpperMargin InterpolationUpperMargin;
+    typedef typename GetMargin<ParticlePush>::LowerMargin ExtendLowerMargin;
+    typedef typename GetMargin<ParticlePush>::UpperMargin ExtendUpperMargin;
 
     typedef PushParticlePerFrame<ParticlePush, MappingDesc::SuperCellSize,
         InterpolationScheme,
         fieldSolver::NumericalCellType > FrameSolver;
 
+    // adjust interpolation area in particle pusher to allow sub-sampling pushes
+    typedef typename PMacc::math::CT::add< InterpolationLowerMargin, ExtendLowerMargin >::type LowerMargin;
```

Comment:
ggrrr aahhhh nooo: We must add the new extended size to https://github.com/ComputationalRadiationPhysics/picongpu/blob/release-0.1.1/src/picongpu/include/fields/FieldB.tpp#L70, and FieldE.tpp else the interpolation for particles gets wrong :-(